src/test/scala/io/iohk/ethereum/consensus/validators/SignedTransactionValidatorSpec.scala

```diff
@@ -18,7 +19,7 @@ class SignedTransactionValidatorSpec extends FlatSpec with Matchers {
 
   val blockchainConfig = BlockchainConfig(Config.config)
 
-  val signedTransactionValidator = new StdSignedTransactionValidator(blockchainConfig)
+  val signedTransactionValidator = new StdSignedTransactionValidator(blockchainConfig, VmConfig(VmMode.Internal, None))
```

Comment:
Could we add tests for IELE? Positive and negative cases

Reply:
@rtkaczyk done